<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vinitas-Air.sthlm.int" ignored="0" name="Test" tests="3" failures="1" timestamp="2022-05-02T16:38:35 CEST" time="7.56" errors="0">
  <testcase name="CreateUser" time="3.476" classname="com.API.tests.RestAPITest"/>
  <testcase name="VerifySingleUserAPI" time="2.135" classname="com.API.tests.RestAPITest">
    <failure type="java.lang.AssertionError" message="expected [2022-05-22] but found [May 22nd 2022]">
      <![CDATA[java.lang.AssertionError: expected [2022-05-22] but found [May 22nd 2022]
at com.API.tests.RestAPITest.VerifySingleUserAPI(RestAPITest.java:47)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- VerifySingleUserAPI -->
  <testcase name="VerifySingleUserwithInvalidResource" time="0.984" classname="com.API.tests.RestAPITest"/>
</testsuite> <!-- Test -->
